brave/brave-core
# Fit custom "welcome" notification text for Linux.

Welcome Brave Ads text with Custom Push notification for Linux:
<img src="https://github.com/brave/brave-core/assets/86598290/5bb3d47d-e704-48bf-b834-ea9bc65e23aa" width="400">


Resolves https://github.com/brave/brave-browser/issues/30605



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Enable Custom Push Brave Ads notification
* Trigger welcome Brave Ads notification
* Make sure that text is not truncated/ellipsized
